templates/repo/view_file.tmpl

```diff
@@ -61,6 +61,9 @@
 				<button class="ui mini basic button unescape-button gt-mr-2 gt-hidden">{{ctx.Locale.Tr "repo.unescape_control_characters"}}</button>
 				<button class="ui mini basic button escape-button gt-mr-2">{{ctx.Locale.Tr "repo.escape_control_characters"}}</button>
 			{{end}}
+			{{if and .ReadmeInList .CanEditReadmeFile}}
+				<a href="{{.RepoLink}}/_edit/{{PathEscapeSegments .BranchName}}/{{PathEscapeSegments .TreePath}}/{{PathEscapeSegments .FileName}}"><span class="btn-octicon" data-tooltip-content="{{ctx.Locale.Tr "repo.editor.edit_this_file"}}">{{svg "octicon-pencil"}}</span></a>
```

Comment:
Maybe it could be simplified to `<a class="octicon-pencil">`? Then no need to use an extra `span`. (if the `span` is a must, ignore my comment)